pkg/azuredisk/nodeserver.go

```diff
@@ -132,9 +132,10 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		fstype = volContextFSType
 	}
 
-	source, lun, err := d.findDiskAndLun(devicePath)
+	// devicePath is actually a lun num
```

Comment:
Since the device is always a lun num, it seems that we can remove the return value `lun` in func `findDiskAndLun`.

Reply:
the in-tree disk driver still uses this function and need lun num, I think we could keep this function now, otherwise we need maintain two different functions.

Reply:
BTW, I will make same fix into k/k when this PR merged.